### DISCUSSION TITLE
Default path variables value to Postman env variables

### DISCUSSION THREAD

**Discussion by timmyGr** [apideck-libraries/portman#683] [Q&A]
So right now the value for path variables is being generating with faker. Wondering how I could set the path variables to default to environment variables globally?

I know I can overwrite them in the config, but that becomes pretty tedious to maintain.

```
  "overwrites": [
    {
      "openApiOperation": "GET::*",
      "overwriteRequestPathVariables": [
        {
          "key": "recordID",
          "value": "{{recordID}}",
          "overwrite": true
        }
      ]
    },
```

**Comment by thim81** [comment]
hi @timmyGr 

Did you find out how to set the default path variables?
Or do need you more assistance?
